.github/workflows/main.yml

```diff
@@ -197,5 +242,6 @@ jobs:
           sleep 30s
       - name: Integration tests
         run: |
+          source ~/venv/bin/activate
```

Comment:
just a reminder to remove the workflow changes from this PR before wrapping up (we'll have to potentially merge #1833 first) when we get a second +1

Reply:
Yes, this PR will wait for #1833. I will remove workflow changes form this PR, rebase and squash commits.